rfcs/expected_fail_message_tag_for_subtests.md

```diff
@@ -0,0 +1,71 @@
+# RFC 165: Support expected-fail-message in test metadata for subtests
+
+## Summary
+
+Add "expected-fail-message" in test metadata so that we can track how
+a subtest fails.
+
+## Details
+
+In Chromium developers are using baselines to track expected output message
```

Comment:
Can you link an example with a permalink? I expect not everyone knows that a baseline is a -expected.txt file?

Reply:
Done